Week-2/Assignment-1/index.html

```diff
@@ -0,0 +1,96 @@
+    <header class="header">
+        <h1>What is Lorem Ipsum?</h1>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+            industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+            scrambled it to make a type specimen book.</p>
+    </header>
+
+    <nav>
+        <ul class="nav">
+            <li><a href="https://www.lipsum.com/" target="_blank">Home</a></li>
+            <li><a href="https://www.lipsum.com/" target="_blank">Contact Us</a></li>
+            <li><a href="https://www.lipsum.com/" target="_blank">Careers</a></li>
+            <li><a href="https://www.lipsum.com/" target="_blank">About Us</a></li>
+        </ul>
+    </nav>
+
+    <main>
+        <figure>
+            <img src="https://www.w3schools.com/html/pic_trulli.jpg" alt="Trulli">
+            <figcaption>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut
+                labore et dolore magna aliqua."</figcaption>
+        </figure>
+
+
+        <article>
+            <h1>What Does WWF Do?</h1>
+            <article>
+                <header>
+                    <h3>Where does it come from?</h3>
+                </header>
+                <p>Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia, looked up one of the more obscure Latin words, consectetur, from a Lorem Ipsum passage, and going through the cites of the word in classical literature, discovered the undoubtable source.</p>
+            </article>
+            <article>
+                <header>
+                    <h3>Why do we use it?</h3>
+                </header>
+                <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English.</p>
+            </article>
+            <article>
+                <header>
+                    <h3>Where can I get some?</h3>
+                </header>
+                <p>There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected humour, or randomised words which don't look even slightly believable. If you are going to use a passage of Lorem Ipsum, you need to be sure there isn't anything embarrassing hidden in the middle of text.</p>
+            </article>
+        </article>
+    </main>
+    <section>
+        <form>
+            <div class="form_input">
+                <label for="name">Full name:</label>
+                <input type="text" id="name" name="name" placeholder="Full name" required>
```

Comment:
**Suggestion:** Try to order the attributes in alphabetical order.

Reply:
Reordered attributes in alphabetical order